environment.yml

```
@@ -3,15 +3,11 @@ channels:
   - conda-forge
 dependencies:
   - cadquery
-  - moab >=5.5.1[build=nompi_tempest_*]
-  - cad_to_dagmc >=0.9.1
-  - openmc >=0.15.0[build=dagmc_nompi_*]
-  - numpy
-  - scipy
+  - moab>=5.5.1[build=nompi_tempest_*]
+  - ca-certificates
+  - certifi
+  - openssl
+  - openmc>=0.15.0[build=dagmc_nompi_*]
+  - cad_to_dagmc>=0.9.1
   - matplotlib
```

Nitpicky but I prefer the inclusion of `numpy` and `scipy`, and the omission of `ca-certificates`, `certifi`, and `openssl`. I suppose if we wanted to be minimalistic, though, we would preserve the omission of `numpy` and `scipy`, and also omit `matplotlib` below (dependency of OpenMC).
```suggestion
  - moab>=5.5.1[build=nompi_tempest_*]
  - cad_to_dagmc>=0.9.1
  - openmc>=0.15.0[build=dagmc_nompi_*]
  - numpy
  - scipy
  - matplotlib
```
or
```suggestion
  - moab>=5.5.1[build=nompi_tempest_*]
  - cad_to_dagmc>=0.9.1
  - openmc>=0.15.0[build=dagmc_nompi_*]
```

---

I want to do more research before messing with `ca-certificates` and `openssl`. One does not simply change these sort of things.